yb-voyager/src/migassessment/sizing.go

```diff
@@ -451,46 +522,34 @@ Returns:
 	sizeInGB: size of source object in gigabytes
 */
 func bytesToGB(sizeInBytes float64) float64 {
-	return sizeInBytes / (1024 * 1024 * 1024)
+	sizeInGB := sizeInBytes / (1024 * 1024 * 1024)
+	// any value less than a 1 MB is considered as 0
+	if sizeInGB < 0.001 {
```

Comment:
Nit: please move this func `bytesToGB` to `utils.go` file inside `utils` package.